src/views/preview/preview.jsx

```diff
@@ -485,6 +490,9 @@ class Preview extends React.Component {
 
 Preview.propTypes = {
     assetHost: PropTypes.string.isRequired,
+    authorId: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Comment:
I see why now, but this still feels more confusing than correct, even though it's technically correct... but fixing it in `mapStateToProps` might be equally confusing (I think you could add `|| null` to the end of all the assignments leading up to `authorId` and `authorUsername`).

Would you please add a short comment to clarify here?

```suggestion
    // If there's no author, this sometimes ends up being `false`
    authorId: PropTypes.oneOfType([PropTypes.string, PropTypes.bool]),
```

Reply:
Done